app/controllers/api/v1/targets_controller.rb

```diff
@@ -11,12 +11,20 @@ def create
         @target = targets.create!(target_params)
       end
 
+      def destroy
+        target.destroy!
```

Comment:
It'd be nice to return a message like `Target was successfully deleted` or something similar